paket.dependencies

```diff
@@ -13,10 +13,12 @@ nuget System.Net.NetworkInformation
 nuget jose-jwt
 
 nuget Fable.Core
-nuget Fable.Elmish.Browser
-nuget Fable.Elmish.Debugger
-nuget Fable.Elmish.React
-nuget Fable.Elmish.HMR
+nuget Fable.React 3.0.0-alpha-002
```

Comment:
@alfonsogarciacaro any timeline on when these things get released?

Reply:
The Fable.React 3 _stablish_ version? At the end we're not going to touch more the API so I guess we can do it now ([this PR](https://github.com/fable-compiler/fable-react/pull/63#issuecomment-370573808) is pending though). I'll have to make some minor but potentially breaking changes for Fable 2, but we can bump another major version then.

Reply:
For the PR you linked, I think a new nuget should be created.

Bumping the major version for Fable 2, is ok no need to hold the prerelease for it I think.